requirements.txt

```diff
@@ -1,2 +1,3 @@
 africastalking==1.1.5
+phonenumbers==8.10.12
```

Comment:
I think in DOSH we have the light phonenumbers library. phonenumberslite==8.10.0
Check if we really need the full library

Reply:
Changed to lite lib